core/core.go

```diff
@@ -9,7 +9,7 @@ import (
 	"time"
 	"unsafe"
 
-	"github.com/kislerdm/chatgpt-c4/core/go-zopfli/zopfli"
+	"github.com/foobaz/go-zopfli/zopfli"
```

Comment:
**(issue)** Please change it to `github.com/kislerdm/diagramastext/core/go-zopfli/zopfli`, it is a git submodule relying on a fork of `github.com/foobaz/go-zopfli/zopfli`. Thanks!

```suggestion
"github.com/kislerdm/diagramastext/core/go-zopfli/zopfli"
```
Please clone the repo using the command:
```commandline
git clone --recurse-submodules git@github.com:kislerdm/diagramastext.git
```

**(todo)** add the clone instruction to readme